lib/ansible/modules/cloud/openstack/os_quota.py

```diff
@@ -287,6 +287,7 @@
 '''
 
 import sys
+from keystoneauth1 import exceptions
```

Comment:
moved keystoneauth1 into the try: statement for import shade since keystoneauth1 is a dependency of shade.